.github/workflows/tests.yml

```diff
@@ -8,41 +8,50 @@ on:
 
 jobs:
   tests:
+    if: github.event.pull_request.draft == false
     runs-on: ubuntu-20.04
     env:
       HOST_TYPE: ci
       REDIS_QUEUE_HOST: redis
       REDIS_STATE_HOST: redis
     container:
-      image: faabric/base:0.0.4
+      image: faasm/faabric:0.0.9
```

Comment:
Could we use enviroment variables here in some way?

Reply:
I get what you're saying, but AFAICT we would either have to template the GH actions file or set the variables at the repo/ organisation level. Do you have an idea of what we might do?